src/test/java/ca/uhn/fhir/jpa/starter/ExampleServerDstu3IT.java

```diff
@@ -61,7 +68,7 @@ class ExampleServerDstu3IT implements IServerSupport {
   @LocalServerPort
   private int port;
 
-  @BeforeEach
+		@BeforeEach
```

Comment:
Can you reverse this change since it seems to be just whitespace?

Reply:
Done.